code/datums/greyscale/json_configs/garden_gnome.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Like this is a lot of json files for things that aren't items that exist in the codebase, a lot of dead code that might never be used.